docs/source/Plugins.rst

```diff
@@ -157,6 +157,20 @@ plugin (``json``) will be executed before the XUnit plugin (``xunit``).
 If the HTML result plugin is installed and enabled (``html``) it will
 be executed before both JSON and XUnit.
 
+Configuring the plugin execution order
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On some circumstances it may be necessary to change the order in which plugins
+are executed.  To do so, add a ``order`` entry a configuration file section
+named after the plugin type.  For ``job.prepost`` plugin types, the section name
+has to be named ``plugins.job.prepost``, and it would look like this::
+
+  [plugins.job.prepost]
+  order = ['myplugin', 'jobscripts']
```

Comment:
Agreed.  As a matter of fact, we should have a system where command line arguments and configuration file settings merge, and this would be transparent.

Btw, the configuration file support for this is actually optional to fix the issue our users have reported.  With the archive (zip) plugin named as zip_archive, it fixes the issue where HTML reports would not be included in the ZIP file.  But, I think it makes sense to let users configure it by using the configuration file at this point.